components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -258,8 +260,14 @@ public static boolean isValidCaptcha(String reCaptchaResponse) throws CaptchaExc
         try {
             try (InputStream in = entity.getContent()) {
                 JsonObject verificationResponse = new JsonParser().parse(IOUtils.toString(in)).getAsJsonObject();
+                if (log.isDebugEnabled()) {
+                    log.info("reCAPTCHA success:" + verificationResponse.get("success") + ", action:" +
+                            verificationResponse.get("action") + ", score:" + verificationResponse.get("score"));
+                }
```

Comment:
Checking isDebugEnabled for info log? Change the log type to debug. Score will not be available for recpatcha v2. In that case better to print the log after checking the score availability so that we can mention whether v2/v3 captcha enabled.